client/components/Carousel.jsx

```diff
@@ -9,14 +9,9 @@ class Carousel extends React.Component {
       <div>
         <h5>Photos from reviews</h5>
         <ul>
-          {props.reviewData.reviewsArray.map((review) => {
-            if (review.reviewPicture) {
-              return (
-                <li key={review.id}><img alt="review" src={review.reviewPicture} /></li>
-              );
-            }
-            return null;
-          })}
+          {props.allImages.map((image) => (
+            <li key={image}><img alt="review" src={image} /></li>
+          ))}
```

Comment:
I'm no expert but would 'picture from a reviewer' be better alt text?

Reply:
Eslint doesn't let you use picture or image as alt tag since it already knows that by the tag you render. Glad this is one of the biggest issues here haha.